testing/dbtest/pgtap-docker/Makefile

```diff
@@ -0,0 +1,20 @@
+REGISTRY_NAME?=docker.io/hashicorpboundary
+TEST_IMAGE_NAME=pgtap
+PG_VERSION=13
+
+# Before running this target a builder instance needs to be setup, ie:
+#  docker buildx create --driver docker-container --use
+docker-build:
+	docker buildx build \
+		--platform linux/amd64,linux/arm64 \
+		--push \
+		-t $(REGISTRY_NAME)/$(TEST_IMAGE_NAME):$(PG_VERSION) \
+		--build-arg PG_VERSION=$(PG_VERSION) \
+		-f Dockerfile .
```

Comment:
I haven't tried running this as I don't have permission to push to this registry

Reply:
If you exclude the `--push` flag you can test that the images build at least.